src/communications/communications.cpp

```diff
@@ -24,7 +24,9 @@
 
 #if BITPIT_ENABLE_MPI==1
 
+#ifndef _WIN32
 #include <unistd.h>
+#endif
```

Comment:
The header is not needed, you can just removed it (tested on gcc and clang).

Reply:
I think the three last commits are general sanity anyway. Feel free to cherry-pick them to master if you want them to land sooner.

Reply:
I agree that those commits are useful. I will merged them into master.